docs/integrations/pytorch_lightning.rst

```diff
@@ -613,9 +613,29 @@ You may also like these two integrations:
 
     <a href="https://github.com/neptune-ai/neptune-examples/blob/master/integrations/pytorch-lightning/docs/Neptune-PyTorch-Lightning-basic.py" target="_blank">GitHub</a>
 
-.. |forum| raw:: html
+.. |PyTorch| raw:: html
 
-    <a href="https://community.neptune.ai/" target="_blank">forum</a>
+    <a href="https://docs.neptune.ai/integrations/pytorch.html" target="_blank">PyTorch</a>
+
+.. |PyTorch Ignite| raw:: html
```

Comment:
You can drop all links that you added here: pytorch, ingnite, catalyst, skorch. optuna, plotly.

We use sphinx's tags and references, to link between pages.

Reply:
**done**